### PR TITLE
Tentative changes

### PR DESCRIPTION
**Pull request by ls-yannick**
These are the updates for the SOC v2 that I'm unsure about. @jerome-leanspace @FlorianLacour please feel free to make changes to this branch as required.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -2,6 +2,14 @@ terraform {
...
     leanspace = {
       source = "leanspace/leanspace"
+
+    }
+  }
+  cloud {
+    organization = "leanspace"
+
+    workspaces {
+      name = "houston-infrastructure"
```
Comment (ls-yannick) [inline main.tf:12]: I'm not aware what this is for. Do we remove it, or do we replace `houston-infrastructure` with something else?

File: satellites/node.tf [hunk 2]
```diff
@@ -10,10 +10,12 @@ module "satellite_base" {
...
     ]
-    tags = {
+    tags = merge({
       "identifier" : var.identifier,
-      "soc" : ""
-    }
+      "soc" : "",
+      }, (var.identifier == "Saturn" || var.identifier == "Iapetus") ? {
+      "LEAFSPACE-ID" : var.identifier == "Saturn" ? "c01bc01721a7c7ab32097737d0dbc100" : (var.identifier == "Iapetus" ? "c50b53098ee2dfd9489eacc95c128b30" : "")
+    } : {})
```
Comment (ls-yannick) [inline satellites/node.tf:18]: Do we need this? I'm not aware why we would.

File: variable.tf [hunk 3]
```diff
@@ -0,0 +1,30 @@
+locals {
+  ground_stations = {
+    Awarua = {
+      elevation           = 12
+      latitude            = -46.52
+      longitude           = -168.38
+      name                = "Awarua"
+      country             = "New Zealand"
+      leafspaceId         = "b485a02f3fe060052f6dc55d5f003284"
```
Comment (ls-yannick) [inline variable.tf:9]: I think we should put placeholder ids here?

File: satellites/plugins/satellite-command-transformer-plugin.jar
Comment (ls-yannick) [inline satellites/plugins/satellite-command-transformer-plugin.jar:1]: Is it ok to ship these files?

File: main.tf (hunk 1 above)
Comment (FlorianLacour) [reply main.tf:12]: For the public we should remove it because they can't connect to terraform cloud

File: satellites/node.tf (hunk 2 above)
Comment (FlorianLacour) [reply satellites/node.tf:18]: For terraform public we should not expose it

File: satellites/plugins/satellite-command-transformer-plugin.jar
Comment (FlorianLacour) [reply satellites/plugins/satellite-command-transformer-plugin.jar:1]: Nop we should remove it

File: variable.tf (hunk 3 above)
Comment (FlorianLacour) [reply variable.tf:9]: yes because it for us and for sandbox env

File: satellites/plugins/satellite-command-transformer-plugin.jar
Comment (ls-yannick) [reply satellites/plugins/satellite-command-transformer-plugin.jar:1]: Is it ok ro remove them and keep the other routes config file as is?

Comment (FlorianLacour) [reply satellites/plugins/satellite-command-transformer-plugin.jar:1]: Yes but it was like this before

**Review by FlorianLacour — APPROVED** [review]

**Pull request merged**